### DISCUSSION TITLE
For each level of hazard severity, capture the likely direct impacts on the asset (for example, “High impact” – catastrophic destruction; “Medium impact” - substantial loss of service and consequent disruption for months; “Low impact”– disruption limited to a few hours only – T #52

### DISCUSSION THREAD

**Discussion by sreyac** [CriticalAssetManagement/CAMS#133] [General]
Need to talk about how this will be implemented

Part of Asset Inventory: Capturing the perils to which each asset is exposed

**Comment by luke-feeney** [comment]
Give a field to input a note on 'total impact' -> likelihood on total loss from loss off asset and impact of time

Time to return -> temporal measure -> can it be returned to acceptable operation within weeks/months/years -> redundancy assessment for specific asset 

Stress testing assets
